### PR TITLE
Y24-109 - Rework Limber documentation for clarity on how SequenceScape & Limber interact and are setup

### LINKED ISSUE #1709: Y24-109 As a team (PSD) we would like to rework the Limber ReadMe so that it is clearer how SequenceScape & Limber interact and are setup.
**Describe the Housekeeping**
As a team (PSD) we would like to rework the Limber ReadMe so that it is clearer how SequenceScape & Limber interact and are setup.

**Acceptance Criteria**
- [x] The Setup instructions should set up one application and then another.
- [x] The Setup instructions should include an explanation of what is being done and why.
- [x] The Readme should clearly indicate that Limber depends on SequenceScape and there is not (hopefully) a two way relationship

**Additional context**
A number of developers have been confused by the existing instructions.
Time box to 2 days.

### PR DESCRIPTION
**Pull request by SHIV5T3R**
Closes #1709 

#### Changes proposed in this pull request

- Reordered the steps for setting up Limber so that one application is setup before another, clearly outlining how many terminals are needed.
- Added some detail on what some of the steps and rake tasks do.

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

**Comment by SHIV5T3R** [comment]
I've changed the order of the steps so it's a little bit easier to figure out how to set things up. Any improvements needed @KatyTaylor @andrewsparkes

File: README.md [hunk 1]
```diff
@@ -39,56 +39,62 @@ Limber should be accessible via [http://localhost:3001](http://localhost:3001).
 
 ## Initial Setup (using native installation)
 
-Steps must be followed in either the Sequencescape repository or this Limber
-repository, as indicated:
+Because Limber relies on SequenceScape as a backend, SequenceScape and it's jobs can be started first. SequenceScape **will** work standalone without Limber.
```
Comment (andrewsparkes) [inline README.md:42]: 'must' be started first.
Limber **will not** work independently.
Limber is the front end for Sequencescape and handles pipeline tracking functionality used by Lab staff.

File: README.md [hunk 2]
```diff
@@ -39,56 +39,62 @@ Limber should be accessible via [http://localhost:3001](http://localhost:3001).
...
-   bundle install
+   bundle exec rails s
    ```
 
-1. In Limber, install the yarn dependencies:
+1. Open a **second** SequenceScape terminal, and configure all data required for Limber:
 
    ```shell
-   yarn install
+   bundle exec rake limber:setup
```
Comment (andrewsparkes) [inline README.md:61]: See` lib/tasks/application.rake`
Line 13
`task post_deploy: %w[record_loader:all limber:setup]`

limber:setup is now done as part of post_deploy. It used be separate.
So you can remove this line.

**Review by andrewsparkes — CHANGES_REQUESTED** [review]
Looks good other than clarifying the SS vs Limber relationship at the top, and that running limber:setup separately is now not required.

I did wonder about troubleshooting tips. e.g. about node versions and yarn, deleting node_modules directory and or assets:clobber. But maybe troubleshooting should be a separate section.

**Review by andrewsparkes — APPROVED** [review]

**Pull request merged**